docs/en/contributor/Contributor-Getting-Started.md

```diff
@@ -119,7 +119,7 @@ All new contributors are recommended to resolve one and only one **New Contribut
 larger tasks. This is a good way to familiarize yourself with the entire process of contributing to
 the Alluxio project.
 
-Browse any of the open [New Contributor Alluxio Tasks](https://github.com/Alluxio/new-contributor-tasks/issues)
+Browse any of the open [New Contributor Alluxio Tasks](https://github.com/Alluxio/alluxio/labels/task-good-first-issue)
```

Comment:
give them info about different categories as well?
the different labels (task-good-first-issue, task-easy, task-medium, task-hard)
otherwise they may be scared away by the thousands of issues lol

Reply:
this would point then to only issues with task-good-first-issue